tests/support_server/setup.pm

```diff
@@ -130,6 +130,9 @@ sub setup_networks {
 
     $setup_script .= "FIXED_NIC=`grep $net_conf->{fixed}->{mac} /sys/class/net/*/address |cut -d / -f 5`\n";
     $setup_script .= "iptables -F\n";
+    $setup_script .= "iptables -A INPUT -i \$FIXED_NIC -p tcp -m tcp  -j ACCEPT\n";
+    $setup_script .= "iptables -A INPUT -i \$FIXED_NIC -p udp -m udp  -j ACCEPT\n";
+    $setup_script .= "iptables -A OUTPUT -p tcp -m state --state NEW -j ACCEPT\n";
```

Comment:
Don't we want to allow all outbound traffic?

Reply:
true. i will fix

Reply:
http://aquarius.suse.cz/tests/3115
http://aquarius.suse.cz/tests/3113